source/evaluate_dataset.py

```diff
@@ -138,10 +139,10 @@ def match_reference(reference, points):
     """
     assert reference.shape == points.shape
     reference_center = np.mean(reference, axis=1)
-    reference -= reference_center[:, None]
+    reference_centered = reference - reference_center[:, None]
     rotation_center = np.mean(points, axis=1)
     points -= rotation_center[:, None]
-    rotation, e = sp.linalg.orthogonal_procrustes(points.T, reference.T)
+    rotation, e = linalg.orthogonal_procrustes(reference_centered.T, points.T)
```

Comment:
Was the order here simply wrong?

Reply:
Yep, but if you wanted to just inverse the rotation then the centres were wrong too - something that wouldn't be visible for a centred dataset. I think. At least that's why I think we didn't notice :D